### PR TITLE
Refactoring project code and removed redundant files

### PR DESCRIPTION
**Pull request by lego963**
Close: #151 
Close: #73

### CODE REVIEW

File: scenarios/core/create_inventory.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def parse_params():
 
 
 def read_state(state_file) -> dict:
-    """Load Terraform state from tfstate file to dict"""
+    """Load Terraform state from tfstate file to dict."""
```
Comment (outcatcher) [inline scenarios/core/create_inventory.py]: One-line comment doesn't require full stop in the end

File: scenarios/core/get_key.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def requires_update(file_name, remote_md5):
 
 
 def get_key_from_s3() -> str:
-    """Download existing key from s3 or create a new one and upload"""
+    """Download existing key from s3 or create a new one and upload."""
```
Comment (outcatcher) [inline scenarios/core/get_key.py]: One-line comment doesn't require full stop in the end

File: scenarios/images/parallel_run.py [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 
 def main():
-    """Run packer for all given targets in a parallel"""
+    """Run packer for all given targets in a parallel."""
```
Comment (outcatcher) [inline scenarios/images/parallel_run.py]: One-line comment doesn't require full stop in the end

File: scenarios/scenario2/resources/vm.tf [hunk 4]
```diff
@@ -45,9 +45,9 @@ resource "opentelekomcloud_compute_secgroup_v2" "scn2_public" {
 }
 
 resource "opentelekomcloud_compute_instance_v2" "basic" {
-  name       = "${var.postfix}_server"
-  image_name = var.ecs_image
-  flavor_id  = var.ecs_flavor
+  name      = "${var.postfix}_server"
+  ecs_image = var.ecs_image
```
Comment (outcatcher) [inline scenarios/scenario2/resources/vm.tf]: `image_name` is argument of `opentelekomcloud_compute_instance_v2` module

File: scenarios/scenario4/variables.tf [hunk 5]
```diff
@@ -2,10 +2,10 @@ variable "username" {}
 variable "password" {}
 variable "region" {}
 variable "tenant_name" {}
-variable "default_az" {}
+variable "availability_zone" {}
 variable "domain_name" {}
-variable "default_flavor" {}
-variable "debian_image" {}
+variable "ecs_flavor" {}
+variable "ecs_image" {}
```
Comment (outcatcher) [inline scenarios/scenario4/variables.tf]: should be `bastion_image`

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,25 +1,25 @@
...
+2. Assign the issue to yourself
 3. Create new branch named `feature/<feature-short-desc>` for features and `fix/<issue-short-desc>` for bugs.
     Please stick to `<group>/<short-description>` convention for branch naming (slash separator for groups and dashes replacing spaces)
 4. Push your local changes to remote branch
 5. Create new [pull request](https://github.com/opentelekomcloud-infra/customer-service-monitoring/pulls)
     (*PR*) with clean title, description mentioning your issue:
-    `Implementing #N` for new features, `Fixing #N` or `Fixed #N` for bugfix. Where `N` is your issue number.
-    **Please note that PR can be merged only if all discussions are solved and it has at least 1 approval**
+    `Implementing #N` for new features, `Fixing #N` or `Fixed #N` for the bugfix. Where `N` is your issue number.
+    **Please note that PR can be merged just if all discussions are solved and it has at least 1 approval**
```
Comment (outcatcher) [inline CONTRIBUTING.md]: Should be 'only' here

**Comment by anton-sidelnikov** [comment]
> Maybe move tenant name and project out of tfvars? @anton-sidelnikov

Yes, i agree. We should fill store this variables in env

File: scenarios/scenario3/resources/volumes.tf [hunk 7]
```diff
@@ -1,9 +1,9 @@
 # Create volume
 resource "opentelekomcloud_blockstorage_volume_v2" "volumes" {
-  count             = 3
-  name              = format("vol-%s", var.volume_type[count.index])
-  size              = 2
-  volume_type       = var.volume_type[count.index]
+  count       = 3
```
Comment (outcatcher) [inline scenarios/scenario3/resources/volumes.tf]: Format fcd up

File: scenarios/scenario1/infrastructure.tf [hunk 8]
```diff
@@ -19,19 +19,18 @@ resource "opentelekomcloud_networking_floatingip_v2" "bastion_public_ip" {}
 module "bastion" {
   source = "../modules/bastion"
 
-  debian_image   = var.ecs_image
-  default_flavor = var.ecs_flavor
+  bastion_image  = var.ecs_image
+  ecs_flavor = var.ecs_flavor
```
Comment (anton-sidelnikov) [inline scenarios/scenario1/infrastructure.tf]: not formatted

File: scenarios/scenario3_5/resources/variables.tf [hunk 9]
```diff
@@ -11,6 +11,7 @@ variable "disc_volume" {}
 variable "subnet" {}
 variable "network" {}
 variable "prefix" {}
+variable "postfix" {}
```
Comment (outcatcher) [inline scenarios/scenario3_5/resources/variables.tf]: prefix + postfix

File: scenarios/scenario4/resources/variables.tf [hunk 10]
```diff
@@ -8,12 +8,13 @@ variable "router_id" {}
 variable "bastion_local_ip" {}
 variable "nodes_count" {}
 variable "prefix" {}
-variable "az" {}
+variable "postfix" {}
```
Comment (outcatcher) [inline scenarios/scenario4/resources/variables.tf]: prefix + postfix

File: scenarios/scenario4/terraform.tfvars [hunk 11]
```diff
@@ -1,10 +1,8 @@
...
-bastion_eip      = ""
-loadbalancer_eip = ""
\ No newline at end of file
+region            = "eu-de"
+availability_zone = "eu-de-01"
+postfix           = "scn4"
+ecs_flavor        = "s2.large.2"
+bastion_image     = "Standard_Debian_9_latest"
+host_image        = "KPI_Scn4_host_image"
+bastion_eip       = ""
```
Comment (outcatcher) [inline scenarios/scenario4/terraform.tfvars]: empty EIPs should be removed

**Pull request merged**